.changeset/forty-spiders-tie.md

```diff
@@ -0,0 +1,5 @@
+---
+"@kaizen/components": minor
+---
+
+Add Heading and Paragraph components from kaizen-legacy
```

Comment:
This can be empty, we aren't planning on publishing these until later

Reply:
Ah, sorted